src/app/mapController.js

```diff
@@ -75,13 +86,18 @@ define([
             this.fLayer.on('load', function () {
                 that.fLayer.renderer.symbol.setSize(config.stationSymbolSize);
             });
+            this.fLayer.on('click', function selectFeatureLayerStation(evt) {
+                that.clearStationSelection();
+                that.selectStation(evt.graphic);
+                evt.stopPropagation();
```

Comment:
I don't think so. What default would I be preventing? I just want to stop the click event from bubbling up to the map.